src/test/java/edu/ksu/canvas/attendance/controller/CourseConfigurationControllerITest.java

```diff
@@ -96,7 +101,8 @@ public void saveCourseConfiguration_HappyPath() throws Exception {
                 .param("saveCourseConfiguration", "Save Course Configuration")
                 .param("defaultMinutesPerSession", String.valueOf(expectedDefaultMinutesPerSession))
                 .param("totalClassMinutes", String.valueOf(expectedTotalClassMinutes))
-                .param("simpleAttendance", String.valueOf(expectedSimpleAttendanceValue)))
+                .param("simpleAttendance", String.valueOf(expectedSimpleAttendanceValue))
+                .param("gradingOn", String.valueOf(false)))
                 .andExpect(status().isOk())
                 .andExpect(view().name("forward:/courseConfiguration/" + irrlevantSectionId + "?updateSuccessful=true"));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "forward:/courseConfiguration/" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "?updateSuccessful=true" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)